data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
-
-Need some warnings about per-user data quota. 
-
-Need some warnings about shared-write area.
-
-
-
-RSP Compute Performance
------------------------
-
-Performance during DP0 may not reflect the performance of the final system (see also Q4), and the resources made available to DP0 delegates may not reflect the final user quotas of the operations-era RSP.
-
-How about running cron jobs? (HFC: I don't think we would let delegates run cron. But what is the motivation of wanting to run cron?)
-
-What about abusing resources?
+User data: No parachutes!
+-------------------------
 
-Testing how algorithms will scale: No access to user batch or parallelisation are available in DP0. Resources provided as part of DP0.1 are limited and meaningful scalability testing of algorithms is not practical. 
+When you access the RSP Notebook aspect, you have a home space you can use to store your notebooks, scripts, extra python modules and small files. Other users cannot see the contents of your home space, but you have no expectation of privacy from observatory staff monitoring the systems. If we experience an incident, we will attempt to restore the contents of your homespace. The contents of your homespace will not be deleted without attempting to notify you.
 
+Data in /scratch is not backed up.
 
+Data in /projects is available for use in the situation where you wish to share files with other RSP users. For this data preview, any files in you put in /projects are readable and writeable by all users. Take care you do not delete other people's files.
 
-Butler-Related Warnings
------------------------
+For this data preview there are no file quotas applied, however usage will be monitored and users may be asked to reduce their footprint, either in their home space or shared data spaces. By using the Butler to write data, you are not using disk space but our data facility's object stores; ample capacity is available there and you should use that in favour of keeping large data files on disk.
 
-For DP0.1 or until we have a client/server Butler and signed URLs (DMTN-169,DMTN-176), all users share full read/write without restriction.  Since there is no individual access control, absolutely no use direct sql access will be allowed to any butler registry database.
-Delegates should only access the Butler repository using standard Butler APIs provided in the Rubin software stack.
+For this data preview, there are no write/delete restrictions on the shared user data Butler repository (butler-us-central1-dp01). That means you could delete your data and other user's data. Be very careful when using the Butler prune collection feature and in particular don't use wildcards with that command.
 
-User-added data in DP0.1’s butler repo will not be migrated to DP0.2’s butler repo. 
+You can reduce the risk of accidents by following the convention documented in `DMTN-167 <https://dmtn-167.lsst.io/>`__ and only write to your own ``u/<user>/*`` collections.
```

Comment:
If you don't want to say "will not" be migrated, perhaps "We cannot guarantee that ... will be migrated"?

Reply:
I think as a user I wouldn't know what to do. Either I need to plan to have that data available or I need to exfiltrate it. Anyway I put a statement back in.